src/lightbox/shareImage.js

```diff
@@ -9,11 +9,12 @@ import * as api from '../api';
 import { openLinkEmbedded } from '../utils/openLink';
 import * as logging from '../utils/logging';
 
+// TODO(i18n): Wire up toasts for translation.
 export default async (url: string, auth: Auth) => {
   const tempUrl = await api.tryGetFileTemporaryUrl(url, auth);
 
   if (tempUrl === null) {
-    showToast('Please share the image from your browser');
+    showToast('Share failed. Please try again from your browser.');
```

Comment:
Hmm, here it's definitely not that sharing failed. I'm not sure the existing text is great, but I feel like this version might be more confusing.

What's really happening here is:
* The server couldn't give us a link we can use directly.
* So instead we're sending the user to the file in a browser, and inviting them to use the "share" feature of the browser.
* That's probably not something they expected when they hit "share" -- and when it does open a browser, it's likely not obvious what that has to do with the sharing they wanted to do.
* So we're giving them some kind of indication of "yes, we heard you, you want to share the file -- this is the path we're offering for doing that".